src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -151,4 +163,5 @@ def add_to_nwbfile(
             individual_name=self.subject_name,
             config_file=str(self.source_data["config_file_path"]),
             timestamps=self._timestamps,
+            container_name=container_name,
```

Comment:
```suggestion
            pose_estimation_container_kwargs=dict(name=container_name),
```

I exposed it in a bit more of a controlled way when I ported the code